.github/pull_request_template.md

```diff
@@ -1,38 +1,33 @@
-Description
------------
+<!--
+⚠️ Your pull request title should be short, detailed, and understandable for all.
+⚠️ If your pull request fixes an open issue, please link to the issue.
 
-Please explain the changes you made here.
+Ensure you have completed the following before continuing.
+✅ I have read the CONTRIBUTING document.
+✅ I have added tests to cover my changes.
+✅ I updated the documentation, where relevant.
+✅ I used type hints (https://www.python.org/dev/peps/pep-0484/) in function signatures.
+✅ I used Google-style (https://google.github.io/styleguide/pyguide.html#383-functions-and-methods) docstrings for functions.
+✅ I added myself / the copyright holder to the AUTHORS file.
```

Comment:
Let's keep this explicit and not in the comments. Being in the comment, they disappear from the webpage, whereas they're useful so that one can get back to them. The original version contained some useful links that here have been removed, e.g., to CONTRIBUTING_DOCS, please add them back.

Reply:
Addressed in [40daf7b](https://github.com/unitaryfund/mitiq/pull/1315/commits/40daf7b0e9879873c88181dd57ceed80d89c43ed).